recipes/osmesa/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "11.2.2" %}
+
+package:
+  name: osmesa
+  version: {{ version }}
+
+source:
+  url: https://github.com/mesa3d/mesa/archive/mesa-{{ version }}.tar.gz
+  fn: mesa-v{{ version }}.tar.gz
+  sha256: a2edf7ad2589f0c3b5cd3d8f8866193c03bf871a38d5dc27423a03f352abe2de
+
+build:
+  number: 0
+  # feel free to figure it out and submit a PR
+  # instructions here: http://mesa3d.org/install.html
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - toolchain
+
+# doesn't look like there are any tests available...
```

Comment:
You can create something that checks if an expected binary and/or library exists. For example:

https://github.com/conda-forge/hdfeos5-feedstock/blob/master/recipe/meta.yaml#L28